### PR TITLE
Intermediate merge: Replace getters in data_objects with properties

### PR DESCRIPTION
**Pull request by GerardvSchie**
Getters are now set as properties in the data_object classes
The __eq__ etc still has to come. That is why this is an intermediate merge

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -294,7 +294,10 @@ good-names=i,
            y0,
            y1,
            z0,
-           z1
```
Comment (saberzero1) [inline .pylintrc:297]: I thought this would be done with regex?

**Review by saberzero1 — DISMISSED** [review]

**Review by saberzero1 — APPROVED** [review]

File: CameraProcessor/processor/utils/text.py [hunk 2]
```diff
@@ -8,12 +8,15 @@
...
-def feature_map_to_json(feature_map=None, object_id=None) -> json:
+def feature_map_to_json(feature_map=None, object_id=None):
     """Sends a featuremap to the orchestrator.
 
     Args:
         feature_map ([Float]): An array of numerical values.
         object_id (Int): The object where the feature_map refers too.
+
+    Returns:
+        json: Json message of the featuremap.
```
Comment (Marieke-van-der-Plaat) [inline CameraProcessor/processor/utils/text.py:19]: Return type van de json zou nog wel mooi zijn

Comment (Marieke-van-der-Plaat) [reply CameraProcessor/processor/utils/text.py:19]: (in de docstring)

File: CameraProcessor/processor/utils/text.py [hunk 3]
```diff
@@ -33,7 +36,7 @@ def bounding_boxes_to_json(bounding_boxes, timestamp) -> json:
     Returns:
         json: JSON representation of the object.
```
Comment (Marieke-van-der-Plaat) [inline CameraProcessor/processor/utils/text.py:37]: Hier zou de type van json ook  nog wel mooi zijn (in docstring)

**Review by Marieke-van-der-Plaat — APPROVED** [review]

**Pull request merged**